swift/example-qr-generator/QrUITests/QrUITestsLaunchTests.swift

```diff
@@ -0,0 +1,32 @@
+//
+//  QrUITestsLaunchTests.swift
```

Comment:
I assume this tests are for sample app. Its sample so no need to have tests. Can we remove it?

Reply:
Done